elastic/kibana
# [OBS-UX-MNGMT] Enrich the alert flyout with a new overview tab.

## Summary
It closes https://github.com/elastic/kibana/issues/173139 by enriching the AlertFlyout for the Observability rules.

## How to test the PR
- Create one rule for each rule type from the list under observability and ensure the rule fires alerts
- Open the alert flyout and check the info is there and well-formatted

## Knowing issues 🐞: 
- ~~Missing formatting for Inventory threshold for selected Metric, e.g. CPU, Memo~~ ✅ 
- ~~Missing formatting for APM threshold for Latency and Error rate~~ ✅
- ~~Missing tests~~ ✅ 
- ~~Tested on SLO~~✅  
- Couldn't tested on Anomaly detection, and APM anomaly 

Here are some screenshots:
![Screenshot 2024-03-27 at 14 12 01](https://github.com/elastic/kibana/assets/6838659/fd893c1a-2890-4da7-b76f-dc2d0d19f054)
![Screenshot 2024-03-27 at 14 11 46](https://github.com/elastic/kibana/assets/6838659/b80b4e33-66b7-42d4-af89-c9efd0531720)
![Screenshot 2024-03-27 at 14 11 26](https://github.com/elastic/kibana/assets/6838659/2a814dd7-09a9-4cb9-a5ae-6fd7c80b4e70)

![Screenshot 2024-03-27 at 14 13 57](https://github.com/elastic/kibana/assets/6838659/368b57ee-37d8-4644-8c4c-8c7c7c5784ad)
![Screenshot 2024-03-27 at 14 12 32](https://github.com/elastic/kibana/assets/6838659/835ca10b-c9bb-452e-a874-930006df62c1)
